contrib/gcppubsub/config/manager.yaml

```diff
@@ -0,0 +1,57 @@
+# This is the control plane manager (reconciler + webhook) for the
+# GCPPubSub event source.
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: gcppubsub-controller-service
+  labels:
+    control-plane: gcppubsub-controller
+spec:
+  selector:
+    control-plane: gcppubsub-controller
+  ports:
+  - port: 443
+---
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
Can be a standard `Deployment`.

Reply:
If we end up scaling horizontally beyond 1 replica, I think the controllers would be stateful in that each one would cover a different range of the keyspace.